packages/combobox/src/Chip/Chip.styles.ts

```diff
@@ -9,18 +9,24 @@ import { ComboboxSize } from '../Combobox.types';
  * Vertical padding on a chip (in px)
  */
 export const chipWrapperPaddingY = {
+  [ComboboxSize.XSmall]: 1,
+  [ComboboxSize.Small]: 0,
   [ComboboxSize.Default]: 2,
   [ComboboxSize.Large]: 4,
 } as const;
 
 /**
  * Height of the chip element (in px)
+ * lineHeight + (2 * paddingY)
  */
 export const chipHeight: Record<ComboboxSize, number> = {
+  [ComboboxSize.XSmall]: 16 + 2 * chipWrapperPaddingY[ComboboxSize.XSmall], // 16 + (2 * 1) = 18
+  [ComboboxSize.Small]:
+    typeScales.body1.lineHeight + 2 * chipWrapperPaddingY[ComboboxSize.Small], // 20 + (2 * 0) = 20
   [ComboboxSize.Default]:
-    typeScales.body1.lineHeight + 2 * chipWrapperPaddingY[ComboboxSize.Default], // 20
+    typeScales.body1.lineHeight + 2 * chipWrapperPaddingY[ComboboxSize.Default], // 20 + (2 * 2) = 24
   [ComboboxSize.Large]:
-    typeScales.body2.lineHeight + 2 * chipWrapperPaddingY[ComboboxSize.Large], // 28
+    typeScales.body2.lineHeight + 2 * chipWrapperPaddingY[ComboboxSize.Large], // 28 + (2 * 4) = 36
 };
```

Comment:
If these lineHeight values were also put in `const`s, we could turn a lot of the copy-pasta'd style logic into functions - I'll refer to this in other comments.